grakn-graph/src/main/java/ai/grakn/graph/internal/AbstractGraknGraph.java

```diff
@@ -121,12 +122,8 @@ public AbstractGraknGraph(G graph, String keyspace, String engine, boolean batch
 
         localIsOpen.set(true);
 
-        if(initialiseMetaConcepts()) {
-            try {
-                commit();
-            } catch (GraknValidationException e) {
-                throw new RuntimeException(ErrorMessage.CREATING_ONTOLOGY_ERROR.getMessage(e.getMessage()));
-            }
+        if(initialiseMetaConcepts() && !(graph instanceof TinkerGraph)) {
+            commitTransaction();
```

Comment:
In the long run, we might want a method to ask what features the graph supports (I think Tinkerpop does a similar thing).